src/static/images/icons/bug-icon.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you'll want to run this through svgo as I don't think we do it in our build steps.

Reply:
Good shout, done.